docs/existing_data.rst

```diff
@@ -0,0 +1,35 @@
+.. _existing_data:
+
+Convert a Standalone to a Patroni Cluster
+=========================================
+
+This section describes the process for converting a standalone PostgreSQL instance into a Patroni cluster.
+
+To deploy a Patroni cluster without using a pre-existing PostgreSQL instance, see :ref:`text <link>` instead.
+
+Procedure
+---------
+
+A Patroni cluster can be started with a data directory from a single-node PostgreSQL database. This is achieved by following closely these steps:
+
+#. Manually start PostgreSQL daemon
+#. Create Patroni super-user as defined in the :ref:`authentication <postgresql_settings>` section of the Patroni configuration. If this user is created in SQL, the following queries achieve this:
+
+.. code-block:: sql
+  CREATE USER $PATRONI_SUPERUSER_USERNAME SUPERUSER CREATEDB CREATEROLE REPLICATION BYPASSRLS;
+  ALTER USER $PATRONI_SUPERUSER_USERNAME ENCRYPTED PASSWORD '$PATRONI_SUPERUSER_PASSWORD';
+
+#. Start Patroni (e.g. ``patroni /etc/patroni/patroni.yml``). It automatically detects that PostgreSQL daemon is already running but its configuration is out-of-date.
+#. Ask Patroni to restart the node with ``patronictl restart cluster-name node-name``.
+
+
+FAQ
+---
+
+#. During Patroni startup, Patroni complains that it cannot bind to the PostgreSQL port.
+
+You need to verify ``listen_addresses`` and ``port`` in ``postgresql.conf`` and ``postgresql.listen`` in ``patroni.yml``. Don't forget that ``pg_hba.conf`` should allow such access.
+
+#. After asking Patroni to restart the node, PostgreSQL displays the error message ``could not open configuration file "/etc/postgresql/10/main/pg_hba.conf": No such file or directory``
```

Comment:
In general, it also might have a meaning that configuration is wrong. It is quite common that postgresql.conf and pg_hba.conf are located in the special config directory (/etc/postgresql/...).
Depending on how you want to manage postgres configuration (with or without Patroni) you can specify either `postgresql.config_dir` to point to the `/etc/postgresql/...` (exact place where postgresql.conf and pg_hba.conf are located) or specify *.conf files in the `postgresql.parameters.hba_file` and `postgresql.custom_conf`. 

You can find a bit more details [here](https://github.com/zalando/patroni/blob/master/docs/SETTINGS.rst#postgresql)